src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,111 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { loadMovie } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState(null);
+  const [isFindMovie, setIsFindMovie] = useState(false);
+
+  const handleChange = (event) => {
+    setQuery(event.target.value);
+    setIsFindMovie(false);
+  };
+
+  const findMovie = () => {
+    if (query === '' || query.length === 0) {
+      return;
+    }
+
+    loadMovie(query).then((response) => {
+      if (response.Response === 'False') {
+        setIsFindMovie(true);
+        setMovie(null);
+
+        return;
+      }
+
+      setMovie({
+        title: response.Title,
+        description: response.Plot,
+        imgUrl: response.Poster,
+        imdbUrl: `https://www.imdb.com/title/${response.imdbID}`,
+        imdbId: response.imdbID,
+      });
+
+      setQuery('');
+    });
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  const addToMovies = () => {
+    if (!movie) {
+      return;
+    }
+
+    addMovie(movie);
+  };
+
+  return (
+    <>
+      <form className="find-movie" onSubmit={event => event.preventDefault()}>
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classNames('input', { 'is-danger': isFindMovie })}
+              value={query}
+              onChange={event => handleChange(event)}
+            />
+          </div>
+
+          {isFindMovie && (
```

Comment:
```suggestion
          {isFindMovie && (
```
Actually, this variable is responsible to show error if movie is not found, so it should be called by opposite name - `isMovieNotFound` or smth like this